### PR TITLE
Add custom roles to organization management

### PR DESCRIPTION
**Pull request by lcawl**
This PR:

- updates pages under https://www.elastic.co/docs/current/serverless/general/manage-access-to-organization to mention custom roles
- replaces an outdated screenshot and removes outdated tab labels
- removes the Observability icon from the custom roles page, since the feature will no longer be applicable to that project type
- adds info about the "Assign roles" button in the custom roles page

NOTE: The information about the custom role feature will not be displayed until the "serverlessCustomRoles" feature flag is changed to "true" in a subsequent PR.

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-1e23u6w4g-elastic-dev.vercel.app

* Built with commit 6df38b6d04943a9884b83d403333336ec3637d1c

> Issues? Visit #next-docs in Slack

File: serverless/pages/manage-access-to-org-user-roles.mdx [hunk 1]
```diff
@@ -24,24 +24,37 @@ You can set a role:
...
+- globally, for all projects of the same type (((es-serverless)), ((observability)), or ((security))). In this case, the role will also apply to new projects created later.
+- individually, for specific projects only. To do that, you have to set the **Role for all** field of that specific project type to **None**.
+
+For example, you can assign a user the developer role for a specific ((es-serverless)) project:
+
+<DocImage size="l" url="../images/individual-role.png" alt="Individual role" />
+
+{/* <DocIf condition={"((serverlessCustomRoles))" === "true"}> */}
+You can also optionally <DocLink slug="/serverless/custom-roles">create custom roles in a project</DocLink>.
+To assign these roles to users, select them from the list of individual instance access roles.
```
Comment (alxchalkias) [inline serverless/pages/manage-access-to-org-user-roles.mdx:38]:
```suggestion
To assign a custom role to users, go to "Instance access roles" and select it from the list under the specific project it was created in.
```

**Review by alxchalkias — COMMENTED** [review]
One suggestion to make it clear that each custom role refers to a specific project

**Review by alxchalkias — APPROVED** [review]
LGTM! Thank you

**Review by kilfoyle — APPROVED** [review]
LGTM! 🦕

**Pull request merged**